server/src/main/java/org/spine3/server/entity/DefaultEntityStorageConverter.java

```diff
@@ -60,12 +60,11 @@ private DefaultEntityStorageConverter(Repository<I, E> repository, FieldMask fie
     protected Tuple<I> doForward(E entity) {
```

Comment:
Is the content of the `Tuple` always defined (e.g. ID -> `EntityRecord`)? If so, let's rename it to make more understandable.